tests/organism_names.tsv

```diff
@@ -1,4 +1,4 @@
-gene_1781	bacteria
-gene_1887	metagenome
-gene_2441	archaea
-genes_3987_3988	cellular organisms
+gene_1781	Bacteria;Proteobacteria;

```

Comment:
The superfluous `;` at the end of the taxonomy are here for test?

Because, I think it could be interesting in this file to have an organism without this superfluous ';' (like `metagenome`) to show that it works with all these different inputs in the same file.

Reply:
Ok. Would you like me to add multiple aother tests ? (I did them for myself but I can add other cases)
- no ";" at the end of line: Bacteria;Proteobacteria
- multiple ";" at the end : Bacteria;Proteobacteria;;;;
- last element does not exist (error) Bacteria;Proteobacter (should get Bacteria)
Thank you for your answer

Reply:
Yes, these three examples are perfect to show the behavior of the tool. And I think you do not have to create new tests. A modification of the `organism_names.tsv` should be enough.  Maybe something like:

| gene_1781             | Bacteria;Proteobacteria                           |
|---------------------------|------------------------------------------------------|
| gene_1887             |        metagenome                                    | 
| gene_2441             | Archaea;Euryarchaeota;Archaeoglobi;;  |
| genes_3987_3988 | Bacteria;Proteobacter                             |

should cover all your example and keep the old example?

Reply:
It seems to cover all. I'll do that as soon as possible.